nservicebus/recoverability/lru-memory-consumption.md

```diff
@@ -10,13 +10,13 @@ tags:
 reviewed: 2019-08-16
 ---
 
-MSMQ and SQL Server transport need to cache exceptions in memory for immediate retries. Therefore, exceptions with a large memory footprint can cause high memory usage of the NServiceBus process. The recoverability cache might require more resources when scaling out endpoints. This page describes how to solve issues caused by excessive memory consumption due to recoverability's caching mechanism.
+MSMQ and SQL Server transport needs to cache exceptions in memory to allow transactions to be cleared before executing recoverability policies. Therefore, exceptions with a large memory footprint can cause high memory usage of the NServiceBus process. Further, the recoverability cache might hold on to items longer when endpoints are scaled out. This page describes how to solve issues caused by excessive memory consumption due to this caching.
```

Comment:
I think `need` would be correct as we're mentioning two transports here (e.g. "they need").

Reply:
I think I'd use `Furthermore` instead of `Further` but that seems to be a detail.

Reply:
Agree, updated (even though Further and Furthermore seems to be interchangeable https://fandom-grammar.livejournal.com/87140.html)